x-pack/legacy/plugins/spaces/public/views/management/components/advanced_settings_title/advanced_settings_title.tsx

```diff
@@ -6,28 +6,38 @@
 
 import { EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Space } from '../../../../../common/model/space';
 import { SpaceAvatar } from '../../../../components';
 
 interface Props {
-  space: Space;
+  getActiveSpace: () => Promise<Space>;
```

Comment:
I want to simplify this once more of this moves to NP. We currently have to [await the NP shim setup](https://github.com/elastic/kibana/pull/40856/files#diff-78f9e6cf751c3d4f8cc259af745d9832R49), which complicates rendering.